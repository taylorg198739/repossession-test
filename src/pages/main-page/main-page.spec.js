import React from 'react'
import { MainPage } from './'
import renderer from 'react-test-renderer';

it('it renders correctly', () => {
    const listMockData = [{
        userId: 6,
        id: 102,
        title: "sed ab consequatur",
        completed: false
    }, {
        userId: 6,
        id: 103,
        title: "sed ab consequatur",
        completed: false
    }, {
        userId: 6,
        id: 104,
        title: "sed ab consequatur",
        completed: false
    }, {
        userId: 6,
        id: 105,
        title: "sed ab consequatur",
        completed: false
    }]
    const MainSnap = renderer.create(<MainPage listData={listMockData} />).toJSON();
    expect(MainSnap).toMatchSnapshot();
})