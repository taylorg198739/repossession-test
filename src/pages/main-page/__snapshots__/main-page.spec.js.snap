// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="d-flex flex-column"
>
  <a
    href="/list-detail/102"
    onClick={[Function]}
  >
    102
     
    sed ab consequatur
  </a>
  <a
    href="/list-detail/103"
    onClick={[Function]}
  >
    103
     
    sed ab consequatur
  </a>
  <a
    href="/list-detail/104"
    onClick={[Function]}
  >
    104
     
    sed ab consequatur
  </a>
  <a
    href="/list-detail/105"
    onClick={[Function]}
  >
    105
     
    sed ab consequatur
  </a>
</div>
`;
